components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -168,12 +170,20 @@ - (void)viewWillLayoutSubviews {
 
   CGSize bannerViewSize = [self.bannerView sizeThatFits:self.view.bounds.size];
   // Adjust bannerViewContainer's frame
-  CGFloat topAreaInset = 0.0f;
+  CGFloat yOrigin = 0.0f;
   if (@available(iOS 11.0, *)) {
-    topAreaInset = self.view.safeAreaInsets.top;
+    yOrigin = self.view.safeAreaInsets.top;
+  } else {
+    if ([self.parentViewController isKindOfClass:[MDCAppBarContainerViewController class]]) {
+      MDCAppBarContainerViewController *containerViewController =
+          (MDCAppBarContainerViewController *)self.parentViewController;
+      UIApplication *safeApplication = [UIApplication mdc_safeSharedApplication];
+      yOrigin = CGRectGetMaxY(containerViewController.appBarViewController.navigationBar.frame) +
+                CGRectGetHeight(safeApplication.statusBarFrame);
+    }
```

Comment:
Pre-iOS 11 you should be able to make use of self.topLayoutGuide to achieve this same effect without relying on assumptions about the AppBar.

Reply:
Good call! I knew there was a pre-iOS 11 API for that, but all my guesses about its name were wrong :)